tools/deploy_prow.sh

```diff
@@ -0,0 +1,56 @@
+#!/usr/bin/env bash
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+# Specific to Prow instance
+CLUSTER="falco-prow"
+ZONE="eu-west-1"
+
+function main() {
+  echo "Getting Kubeconfig for cluster access" 
+  updateKubeConfig
+  echo "Launching Configmaps, and prereq software" 
+  launchConfig
+  echo "Launching Prow microservices" 
+  launchProw
+  echo "All done!"
+}
+
+function updateKubeConfig() {
+  aws eks --region ${ZONE} update-kubeconfig --name ${CLUSTER}-test-infra
+}
+
+function launchConfig(){
```

Comment:
Why this script does not use the already existing makefile commands? 

I would stick with one single strategy for deploying it: bash or make commands

Reply:
The idea was that the scripts were for CircleCI, and the make commands were quick hacks locally for testing. Future we could move all to make, but it would require to testing.

Reply:
Can't we reuse the make commands in that bash script?

Reply:
We can, except in the case we need to remake the entire cluster the commands will fail since are replacement commands.  `kubectl create secret generic hmac-token --from-literal=hmac="$$(./tools/1password.sh -d config/prow/hmac-token)" --dry-run -o yaml | kubectl replace secret hmac-token -f -`

Reply:
Moving it all to make based for CircleCI we can try in another PR, would require some time for testing, WDYT?

Reply:
👌